packages/libs/react-ui/src/components/Footer/FooterLinkItem.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  colorVariants,
+  linkBoxClass,
+  linkClass,
+  spanClass,
+} from './Footer.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export type Target = '_self' | '_blank';
+export interface IFooterLinkItemProps {
+  title: string;
+  href?: string;
+  target?: Target;
+  color?: keyof typeof colorVariants;
+}
+
+export const FooterLinkItem: FC<IFooterLinkItemProps> = ({
+  title,
+  href,
+  target = '_blank',
+  color = 'default',
+}) => {
+  const classLinkList = classNames(linkClass, colorVariants[color]);
+  const classSpanList = classNames(spanClass, colorVariants[color]);
+  return (
+    <div className={linkBoxClass} data-testid="kda-footer-link-item">
+      {href !== undefined ? (
+        <a className={classLinkList} href={href} target={target}>
```

Comment:
how will this work with Next/Links?
most apps will use Next. and just using an a tag will navigate away from the app (not SPA, I mean)

Reply:
I think, if you refactor like this it can work.
put the a tag as a child of the FooterLinkItem.

```
<FooterLinkItem>
  <Link href="#" passHref legacyBehavior>
    <a>link title</a>
  </Link>
</FooterLinkItem>
```